admin/includes/configuration_checks.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @package admin
+ * @copyright Copyright 2015 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version 
+ */
+
+  // Function used for configuration checks only
+  function check_configuration($variable, $check_string) { 
+     // Expected format of $check string is a json encoded array 
+     // with the following paramters: 
+     // error: defined constant with error message
+     // id: id of the filter to apply.   
+     // options: per http://php.net/manual/en/function.filter-var.php
```

Comment:
The preferred modern-day approach to putting documentational comments in code is to use DocBlock syntax, above the function, instead of inside it.